SteamKit2/Tests/KeyValueFacts.cs

```diff
@@ -500,6 +500,30 @@ public void KeyValuesSavesTextToStream()
             Assert.Equal( expected, text );
         }
 
+        [Fact]
+        public void KeyValuesUnsignedByteConversion()
+        {
+            byte expectedValue = 37;
+
+            var kv = new KeyValue("key", "37");
+            Assert.Equal(kv.AsUnsignedByte(), expectedValue);
```

Comment:
XUnit syntax is `Assert.Equal(expected, actual)`, not `Assert.Equal(actual, expected)`. Same on L512, L521 and L524.